neo/Network/RPC/RpcClientTools.cs

```diff
@@ -0,0 +1,205 @@
+using Neo.Network.P2P.Payloads;
+using Neo.SmartContract.Native;
+using Neo.VM;
+using Neo.Wallets;
+using System;
+using System.Linq;
+using System.Numerics;
+using System.Threading.Tasks;
+
+namespace Neo.Network.RPC
+{
+    /// <summary>
+    /// NEO APIs throught RPC
+    /// </summary>
+    public class RpcClientTools
```

Comment:
Make it as a static class?

Reply:
These methods depend on a ```RpcClient``` instance, this should be provided by the user.

Reply:
I still think that users should use `RpcClient`, `ContractClient`, `Nep5API` and `PolicyAPI` directly.

Reply:
OK, I will change this.

Reply:
WalletAPI is good.